sklearn/neural_network/_multilayer_perceptron.py

```diff
@@ -1263,6 +1282,20 @@ class MLPRegressor(RegressorMixin, BaseMultilayerPerceptron):
     out_activation_ : string
         Name of the output activation function.
 
+    Examples
+    --------
+    >>> from sklearn.neural_network import MLPRegressor
+    >>> from sklearn.datasets import make_regression
+
+    >>> X, y = make_regression(n_samples=20, n_features=5, random_state=1)
```

Comment:
Same for here.

Reply:
@adrinjalali this one is still open

@FollonSaxBass
Did the last suggestions you got make sense to you? Will you be be able to work on them?

Reply:
Oh wow, sorry for the delay (i didn't notice the suggestions), I'll push the fix as soon as possible (during the weekend).